trade-finance-manager-ui/templates/case/facility/_macros/facility_details.njk

```diff
@@ -111,17 +111,17 @@
         <dt class="govuk-grid-column-one-quarter govuk-body-s">Cover end date</dt>
         <dd class="govuk-grid-column-three-quarters govuk-body-s" data-cy="facility-cover-end-date">{{ facility.dates.coverEndDate | dashIfEmpty }}</dd>
 
-        {% if isTfmFacilityEndDateFeatureFlagEnabled %}
+        {% if shouldDisplayFacilityEndDate %}
           <dt class="govuk-grid-column-one-quarter govuk-body-s">Bank has provided a facility end date</dt>
-          <dd class="govuk-grid-column-three-quarters govuk-body-s" data-cy="is-using-facility-end-date">{{ facility.dates.isUsingFacilityEndDate | formatBooleanAsString }}</dd>
+          <dd class="govuk-grid-column-three-quarters govuk-body-s" data-cy="facility-is-using-facility-end-date">{{ facility.dates.isUsingFacilityEndDate | formatBooleanAsString }}</dd>
 
           {% if (facility.dates.isUsingFacilityEndDate === false) %}
             <dt class="govuk-grid-column-one-quarter govuk-body-s">Bank review date</dt>
-            <dd class="govuk-grid-column-three-quarters govuk-body-s" data-cy="bank-review-date">{{ facility.dates.bankReviewDate | formatIsoDateString("d MMMM yyyy") | dashIfEmpty }}</dd>
+            <dd class="govuk-grid-column-three-quarters govuk-body-s" data-cy="facility-bank-review-date">{{ facility.dates.bankReviewDate | formatIsoDateString("d MMMM yyyy") | dashIfEmpty }}</dd>
 
           {% else %}
             <dt class="govuk-grid-column-one-quarter govuk-body-s">Facility end date</dt>
-            <dd class="govuk-grid-column-three-quarters govuk-body-s" data-cy="facility-end-date">{{ facility.dates.facilityEndDate | formatIsoDateString("d MMMM yyyy") | dashIfEmpty }}</dd>
+            <dd class="govuk-grid-column-three-quarters govuk-body-s" data-cy="facility-facility-end-date">{{ facility.dates.facilityEndDate | formatIsoDateString("d MMMM yyyy") | dashIfEmpty }}</dd>
```

Comment:
can the extra facility be removed

Reply:
see above (keeping convention)

Reply:
it doesn't sound right.  I understand about amendment-- but facility-facility doesn't make sense.  If facility-end-date is already taken, another word could be used?

Reply:
Can it be changed to `facility-details-end-date` instead so it's consistent with the file name

Reply:
Ok I've changed it to just `facility-end-date` :)